src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -37,9 +37,12 @@ public DefaultServer(List<Type> typesToInclude)
             configuration.EnableInstallers();
 
             configuration.DisableFeature<TimeoutManager>();
-            configuration.Recoverability()
-                .Delayed(delayed => delayed.NumberOfRetries(0))
-                .Immediate(immediate => immediate.NumberOfRetries(0));
+            if (!endpointConfiguration.PerformDefaultRetries)
```

Comment:
I disagree. Why do we have acceptance tests testing the default retries? Acceptance tests should test that the retries behave according to whatever is configured. If you want to test a certain retry behavior, define the values you want to test, it makes the test more explicit and readable. Do we need acceptance tests to test default values? That seems to be something a unit test should be enough.